src/gc/gcinterface.h

```diff
@@ -546,16 +546,16 @@ class IGCHeap {
     virtual HRESULT Initialize() = 0;
 
     // Returns whether nor this GC was promoted by the last GC.
-    virtual BOOL IsPromoted(Object* object) = 0;
+    virtual bool IsPromoted(Object* object) = 0;
 
     // Returns true if this pointer points into a GC heap, false otherwise.
-    virtual BOOL IsHeapPointer(void* object, BOOL small_heap_only = FALSE) = 0;
+    virtual bool IsHeapPointer(void* object, bool small_heap_only = false) = 0;
```

Comment:
The warning is only issued if you are converting BOOL variable to bool. It is warning you about that there will be one or two extra machine instructions to do this conversion.

The warning is not issues you are converting FALSE/TRUE constant to bool. The compiler will optimize this conversion out, and so it does not bother warning you.

I agree that it would be nice to get the upper case FALSE/TRUE changed to the lower case one.